src/GraphQL/Language/AST/VariableReference.cs

```diff
@@ -18,7 +20,7 @@ public VariableReference(NameNode name)
         /// <summary>
         /// Returns the name of the variable being referenced.
         /// </summary>
-        public string Name => NameNode.Name;
+        public string Name => NameNode.Name!;
```

Comment:
Why does `NameNode.Name` nullable?

Reply:
I don't know.  It isn't now, so maybe I already updated it in a subsequent PR.